packages/main/src/components/AnalyticalTable/types/index.ts

```diff
@@ -329,7 +329,7 @@ export interface AnalyticalTableColumnDefinition {
    * __Note__: You can also specify deeply nested values with accessors like `info.hobby` or even `address[0].street`
    * __Note__: If no `accessor` is set, or the `accessor` is a function, the `id` property has to be set.
    */
-  accessor?: string | ((row: RowType, rowIndex: number) => any);
+  accessor?: string | ((row: Record<string, any>, rowIndex: number) => any);
```

Comment:
It looks like we not only used the wrong type here, but also missed typing some additional parameters:

```suggestion
  accessor?: string | ((originalRow: Record<string, any>, rowIndex: number, row: RowType, allRows: RowType[], data: Record<string, any>[]) => any);
```